test/stored_queries/test_ncbi_tax.py

```diff
@@ -0,0 +1,61 @@
+"""
+Tests for the ncbi taxonomy stored queries.
+"""
+import json
+import unittest
+import requests
+# import time
+
+from test.helpers import get_config
+
+_CONF = get_config()
+
+
+def create_test_docs(ncbi_taxon, ncbi_child_of_taxon):
+    """Create edges and vertices we need for tests."""
+    body = '\n'.join([json.dumps(d) for d in ncbi_taxon])
+    resp = requests.put(
+        _CONF['re_api_url'] + '/api/v1/documents',
+        params={'overwrite': True, 'collection': 'ncbi_taxon'},
+        data=body,
+        headers={'Authorization': 'admin_token'}
+    )
+    if not resp.ok:
+        raise RuntimeError(resp.text)
+    ncbi_taxon_results = resp.json()
+    body = '\n'.join([json.dumps(d) for d in ncbi_child_of_taxon])
+    resp = requests.put(
+        _CONF['re_api_url'] + '/api/v1/documents',
+        params={'overwrite': True, 'collection': 'ncbi_child_of_taxon'},
+        data=body,
+        headers={'Authorization': 'admin_token'}
+    )
+    if not resp.ok:
+        raise RuntimeError(resp.text)
+    ncbi_child_of_taxon_results = resp.json()
+    return {
+        'ncbi_taxon': ncbi_taxon_results,
+        'ncbi_child_of_taxon': ncbi_child_of_taxon_results
+    }
+
+
+class TestNcbiTax(unittest.TestCase):
+
+    def test_valid(self):
+        """Test a valid query."""
+        taxon_docs = [
+            {'_key': '1', 'scientific_name': 'Bacteria', 'rank': 'Domain'},
```

Comment:
not sure if this is a productive comment (probably not), but using 'scientific_name' as the field here is a little confusing since it usually refers to 'Genus species'. not sure thats a big deal but thought I'd bring it up

Reply:
What do you mean by "genus species"?
Do you mean to say that scientific name usually refers to species name?

We're using scientific name to refer to a human-readable name assigned to any rank in the taxonomy, in contrast to the taxonomy_id